### DISCUSSION TITLE
ShaderGlass 0.6 released

### DISCUSSION THREAD

**Discussion by mausimus** [mausimus/ShaderGlass#23] [Announcements]
- fine-tuning of Parameters for each shader
- ability to export a .png screenshot
- RetroArch shaders repo sync (~100 new shader presets)
- new keyboard shortcuts: Save = Ctrl+Shift+S, Pause/Unpause = Ctrl+Shift+P
- custom Aspect Ratio Correction

**Comment by Zuecifer** [comment]
Where would I be able to change the keyboard shortcuts? Is it compiled in the exe or is it in a cpp file by chance?
